xdsl/dialects/builtin.py

```diff
@@ -50,6 +51,7 @@
     var_operand_def,
     var_result_def,
 )
+from xdsl.irdl.irdl import ParamAttrConstraint
```

Comment:
```suggestion
from xdsl.irdl import ParamAttrConstraint
```
That's something I need to fix maybe so the IDEs won't automatically import from there.

Reply:
That would be great because it's kind of tedious to fix by hand. I have done a bit of googling and I haven't found a good option. Maybe an issue for the vscode extension team directly?